osidb/serializer.py

```diff
@@ -717,6 +717,10 @@ class FlawSerializer(
     meta = serializers.SerializerMethodField()
     meta_attr = serializers.SerializerMethodField()
 
+    # This line forces the deprecated "is_major_incident" field NOT to change
+    # from boolean to string. Otherwise, some unknown logic turns it into string.
```

Comment:
Probably because it's a descriptor and it cannot infer what the return type will be, maybe with type hints it can?

Reply:
You may be right, but this solution is also correct and not complicated, so I would prefer it for now. I will check later with @JakubFrejlach if it could be solved as you recommended.